### PR TITLE
CORE-831: Add `cryptoWalletCreateTransferMultiple()`

### PR DESCRIPTION
**Pull request by EBGToo**

### CODE REVIEW

File: WalletKitCore/crypto/BRCryptoWallet.c [hunk 1]
```diff
@@ -581,6 +581,68 @@ cryptoWalletAsGEN (BRCryptoWallet wallet) {
...
+    BRCryptoCurrency currency = cryptoUnitGetCurrency(unit);
+
+    switch (wallet->type) {
+        case BLOCK_CHAIN_TYPE_BTC: {
+            BRWalletManager bwm = wallet->u.btc.bwm;
+            BRWallet *wid = wallet->u.btc.wid;
+            BRAddressParams params = BRWalletGetAddressParams(wid);
+
+            BRTxOutput outputs [specsCount];
+            memset (outputs, 0, specsCount * sizeof(BRTxOutput));
```
Comment (michael-brd) [inline WalletKitCore/crypto/BRCryptoWallet.c]: What happens if specsCount is null?

File: WalletKitCore/crypto/BRCryptoWallet.c [hunk 2]
```diff
@@ -581,6 +581,68 @@ cryptoWalletAsGEN (BRCryptoWallet wallet) {
...
+
+                assert (cryptoWalletGetType(wallet) == cryptoAddressGetType(spec->target));
+                assert (cryptoAmountHasCurrency (spec->amount, currency));
+
+                BRCryptoBoolean isBitcoinAddr = CRYPTO_TRUE;
+                BRAddress address = cryptoAddressAsBTC (spec->target, &isBitcoinAddr);
+                assert (isBitcoinAddr == AS_CRYPTO_BOOLEAN (BRWalletManagerHandlesBTC (bwm)));
+
+                BRCryptoBoolean overflow = CRYPTO_FALSE;
+                uint64_t value = cryptoAmountGetIntegerRaw (spec->amount, &overflow);
```
Comment (michael-brd) [inline WalletKitCore/crypto/BRCryptoWallet.c]: I forget; does this take into account if the amount was bitcoin vs satoshis?

Comment (EBGToo) [reply WalletKitCore/crypto/BRCryptoWallet.c]: The underlying value is always in the integer format; it is only when displaying an Amount that conversion is done to the specified or default unit.

File: WalletKitCore/crypto/BRCryptoWallet.c (hunk 1 above)
Comment (EBGToo) [reply WalletKitCore/crypto/BRCryptoWallet.c]: Addressed.

**Pull request merged**